pytri/js/GraphLayer.js

```diff
@@ -54,7 +54,7 @@ class ColorGraphLayer extends window.substrate.Layer {
 
         this.pSys = particleSystem;
         scene.add(particleSystem);
-
+        window.graph = this.graph;
```

Comment:
Where do we use this? I'd like to not add a global variable to the namespace if possible.

Reply:
Left over from debugging. Good catch!